repeated-integration-tests.sh

```diff
@@ -28,15 +30,16 @@ while [ $runcount -lt $runs ]; do
         exec:exec@stop-jfr-datasource \
         exec:exec@stop-grafana \
         exec:exec@stop-container \
-        exec:exec@destroy-pod |& tee -a $logfile
-    if [ $? -ne 0 ]; then
+        exec:exec@destroy-pod |& tee -a "${client_logfile}"
+    if [ "${PIPESTATUS[0]}" -ne 0 ]; then
         failures=$((failures+1))
     fi
     runcount=$((runcount+1))
+    podman logs cryostat-itest > "${server_logfile}"
 done
 
 echo
 echo "########################"
 echo "Test runs completed"
-echo "Failures: $failures"
-echo "Runs: $runs"
+echo "Failures: ${failures}"
```

Comment:
I see `Failures: 0 ` - am I on the right branch? 
At least the log files aren't appearing in the untracked files.
```
jalaw@localhost ~/cryostat ((3cd5773f…))> git status
HEAD detached at aazores/repeated-itest-script-fixup
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.vscode/

nothing added to commit but untracked files present (use "git add" to track)
```
```
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------< io.cryostat:cryostat >------------------------
[INFO] Building cryostat 2.1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (create-pod) @ cryostat ---
abaadac577085a9f3f75a5b5fabf22e953a11133120fda902e69690a4d0ef209
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (start-jfr-datasource) @ cryostat ---
b2f13434efa9a600a2f69984d70d1b413bbaab6d8dc6288e7569db7bd639297c
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (start-grafana) @ cryostat ---
b702fedc83b0644623d62ca222655191013dc01111efc8e0dc36dc0567bad93d
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (start-container) @ cryostat ---
92f2ca47dabd242e06a8f29c7af301531790a61254fca534973faea294abf3ac
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (wait-for-container) @ cryostat ---
curl: (56) Recv failure: Connection reset by peer
{"dashboardAvailable":true,"cryostatVersion":"v2.0.0-SNAPSHOT-236-gb9bace35-dirty","datasourceAvailable":true}[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.2:integration-test (default-cli) @ cryostat ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------

...

[INFO] Running itest.NoopAuthV2IT
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.01 s <<< FAILURE! - in itest.NoopAuthV2IT
[ERROR] shouldRespond200  Time elapsed: 0.01 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: <{"meta":{"type":"application/json","status":"OKthisshouldfail"},"data":{"result":{"username":""}}}>
     but: was <{"meta":{"type":"application/json","status":"OK"},"data":{"result":{"username":""}}}>
	at itest.NoopAuthV2IT.shouldRespond200(NoopAuthV2IT.java:81)

[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   NoopAuthV2IT.shouldRespond200:81 
Expected: <{"meta":{"type":"application/json","status":"OKthisshouldfail"},"data":{"result":{"username":""}}}>
     but: was <{"meta":{"type":"application/json","status":"OK"},"data":{"result":{"username":""}}}>
[INFO] 
[ERROR] Tests run: 103, Failures: 1, Errors: 0, Skipped: 1
[INFO] 
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-jfr-datasource) @ cryostat ---
jfr-datasource-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-grafana) @ cryostat ---
grafana-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-container) @ cryostat ---
cryostat-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (destroy-pod) @ cryostat ---
d252daa6693a5234c1a3ef0d265fc8ddf4deec41d3f55c7dfbfa04f63fc4b3e0
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:53 min
[INFO] Finished at: 2021-12-02T13:25:28-05:00
[INFO] ------------------------------------------------------------------------
Error: no container with name or ID "cryostat-itest" found: no such container

########################
Test runs completed
Failures: 0
Runs: 1
```

Reply:
That's weird. It worked for me, although perhaps I had a test "Error" execution result and not a "Failure"? It seems weird that Maven's exit status would still be 0 on a Failure though. I'll see if I can reproduce this.

Reply:
Hmm nevermind, a second `git fetch aazores` seemed to fix it.
```
jalaw@localhost ~/cryostat (close-rules-connections)> git fetch aazores
remote: Enumerating objects: 18, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 15 (delta 10), reused 13 (delta 8), pack-reused 0
Unpacking objects: 100% (15/15), 1.95 KiB | 29.00 KiB/s, done.
From github.com:andrewazores/cryostat
 * [new branch]        itest-container-health      -> aazores/itest-container-health
   3cd5773f..3d521e26  repeated-itest-script-fixup -> aazores/repeated-itest-script-fixup
jalaw@localhost ~/cryostat (close-rules-connections)> git checkout repeated-itest-script-fixup
M	src/test/java/itest/NoopAuthV2IT.java
Branch 'repeated-itest-script-fixup' set up to track remote branch 'repeated-itest-script-fixup' from 'aazores'.
Switched to a new branch 'repeated-itest-script-fixup'
```

Reply:
Huh.

```
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   ReportIT.testGetReportShouldSendFile:160 » Execution io.vertx.ext.web.handler....
[INFO] 
[ERROR] Tests run: 103, Failures: 0, Errors: 1, Skipped: 1
[INFO] 
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-jfr-datasource) @ cryostat ---
[INFO] [stdout] jfr-datasource-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-grafana) @ cryostat ---
[INFO] [stdout] grafana-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (stop-container) @ cryostat ---
[INFO] [stdout] cryostat-itest
[INFO] 
[INFO] --- exec-maven-plugin:3.0.0:exec (destroy-pod) @ cryostat ---
[INFO] [stdout] cad3544138cb18f9e36babd33f28f2fba919d15987bb8c9fe0faf645de0a0a65
[INFO] Segment walltime 243 s, segment projects service time 243 s, effective/maximum degree of concurrency 1.00/7
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:03 min (Wall Clock)
[INFO] Finished at: 2021-12-02T14:28:53-05:00
[INFO] ------------------------------------------------------------------------
Error: no container with name or ID "cryostat-itest" found: no such container

########################
Test runs completed
Failures: 0
Runs: 1
```

This did work earlier in the patch series. Not sure what I did to break it again.

Reply:
Should be fixed now, I think.

Reply:
Yes, all fixed!